apps/meteor/ee/server/models/raw/ReadReceipts.ts

```diff
@@ -16,4 +17,92 @@ export class ReadReceiptsRaw extends BaseRaw<ReadReceipt> implements IReadReceip
 	findByMessageId(messageId: string): FindCursor<ReadReceipt> {
 		return this.find({ messageId });
 	}
+
+	removeByUserId(userId: string): Promise<DeleteResult> {
```

Comment:
Do we need an index on userId?

Reply:
Done, I added indexes to `userId` and `roomId` too
cc @sampaiodiego